docs/site/DEVELOPING.md

```diff
@@ -299,6 +299,34 @@ To update dependencies to their latest compatible versions:
 npm run update-all-deps
 ```
 
+### Requiring connectors
```

Comment:
We probably should list this as a **caveat** of the Lerna Monorepo - the symbolic links break Node.js module visibility in some cases.